### PR TITLE
Support flexible installments

### PR DESCRIPTION
**Pull request by voneiden**
Support calculating value for flexible installments based on the sale price of the apartment.

### CODE REVIEW

File: application_form/tests/api/test_apartment_reservation_api.py [hunk 1]
```diff
@@ -209,9 +212,39 @@ def test_root_apartment_reservation_detail_installment_candidates(
...
+        "type": installment_template_4.type.value,
+        "amount": 5555556,
+        "account_number": installment_template_4.account_number,
+        "due_date": None,
+    }
+
 
 @pytest.mark.django_db
-def test_contract_pdf_creation_unauthorized(user_api_client):
+def test_contract_pdf_creation_unauthorized(elasticsearch, user_api_client):
```
Comment (tuomas777) [inline application_form/tests/api/test_apartment_reservation_api.py:247]: All of these elasticsearch fixture additions seem to be completely unrelated to the flexible installment thing, so it would be nicer if these were in their own commit.

**Review by tuomas777 — APPROVED** [review]
Just one nit-pick about commit purity that you could fix IMO as that seems like a pretty trivial thing to do, other than that LGTM, great work! 👍 🚀

File: application_form/tests/api/test_apartment_reservation_api.py (hunk 1 above)
Comment (voneiden) [reply application_form/tests/api/test_apartment_reservation_api.py:247]: Oh yeah, absolutely, I may have squashed this accidentally. I'll separate it. The main point here was that the `setup_elasticsearch` and `teardown_elasticsearch` are skipped without using any ES fixture. Just for the sake of consistency as  the tests will leave the test index dangling without it.
